generate.sh

```diff
@@ -2,14 +2,14 @@
 
 virtualenv --python=python yaydocvenv
 source yaydocvenv/bin/activate
-pip install sphinx
-pip install recommonmark
-pip install pypandoc
 git clone https://github.com/fossasia/yaydoc.git yaydocclone
 mkdir yaydoctemp
 cp -a yaydocclone/scripts/ yaydoctemp/
 cp -a yaydocclone/templates/ yaydoctemp/
+cp -a yaydocclone/fossasia/ yaydoctemp/
+cp -a yaydocclone/fossasia/requirements.txt yaydoctemp/
```

Comment:
requirements.txt should be in yaydocclone/, not within yaydocclone/fossasia/ . Any why is that file being copied. I think you should install the requirements where we are cloning yaydoc